src/sssom/parsers.py

```diff
@@ -264,10 +271,32 @@ def parse_sssom_json(
 ) -> MappingSetDataFrame:
     """Parse a TSV to a :class:`MappingSetDocument` to a  :class`MappingSetDataFrame`."""
     raise_for_bad_path(file_path)
-    metadata = _get_prefix_map_and_metadata(prefix_map=prefix_map, meta=meta)
 
     with open(file_path) as json_file:
         jsondoc = json.load(json_file)
+
+    # Get prefix map from jsondoc and update metadata.
+    # This takes priority over default prefix_map in case of a tie.
+    jsondoc_prefix_map = jsondoc["@context"]
+
+    # Convert keys in both maps to lower case for comparison
```

Comment:
Is this wise? I think we should not be doing all of this lowercasing. The prefix maps should be correct as they are. Doing all of these ad-hoc operations makes sssom-py insanely hard to work with (I can tell you this from first hand experience since I have tried to understand the whole history of the package)